.github/workflows/cc-job-image-base.yaml

```diff
@@ -0,0 +1,97 @@
+name: Build CC-Job-Image-Base
+run-name: Building CC-Job-Image-Base
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    environment: build
+    permissions:
+      contents: write
+      pull-requests: write
+      packages: write
+    env:
+      platforms: linux/amd64,linux/arm64
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Git Identity
+        run: |
+          git config --global --add safe.directory $PWD
+          git config --global user.name 'Gardener-CICD Bot'
+          git config --global user.email 'gardener.ci.user@gmail.com'
+
+      - name: Setup Docker-Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v3
+        with:
+          platforms: ${{ env.platforms }}
+
+      - name: docker-auth
+        uses: docker/login-action@v3
+        with:
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+          registry: ghcr.io
+
+      - name: Build and Publish CC-Job-Image-Base
+        uses: docker/build-push-action@v6
+        with:
+          push: true
+          platforms: ${{ env.platforms }}
+          tags: ghcr.io/${{ github.repository_owner }}/cc-utils/job-image-base:latest
```

Comment:
while it is okay to (also) set `latest`-tag, I would prefer to stick to also setting version-specific tags